### DISCUSSION TITLE
disable onclick table row

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#4885] [Help] [answered]
how can i disable the on mouse over effect on the tables which redirects to the edit page?

**Comment by lutfifirdaus** [comment, marked as answer]
You can add `disable()` function on method `table` 's `actions`  on `Tables\Actions\EditAction` like this `Tables\Actions\EditAction::make()->disable(true)`

**Reply by lawre17** [reply]
I tried this but it didn't work as expected I found this solution later:
`public static function table(Table $table): Table
    {
        return $table
            ->columns([
//your columns here
])->recordUrl(null)`
